packages/core/contracts/zodiac/OptimisticOracleModule.sol

```diff
@@ -0,0 +1,232 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity ^0.8.6;
+
+import "@gnosis.pm/zodiac/contracts/core/Module.sol";
+
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+
+import "../oracle/implementation/Constants.sol";
+import "../oracle/interfaces/FinderInterface.sol";
+import "../oracle/interfaces/SkinnyOptimisticOracleInterface.sol";
+import "../oracle/interfaces/OracleAncillaryInterface.sol";
+import "../common/implementation/Lockable.sol";
+import "../common/interfaces/AddressWhitelistInterface.sol";
+import "../common/implementation/AncillaryData.sol";
+
```

Comment:
if you are going to use NatSpec you should include a `title` comment like this: https://github.com/UMAprotocol/protocol/blob/ceb5a468d6cfb9cf1996fb6ad058ac36d5f7db65/packages/core/contracts/oracle/implementation/Voting.sol#L22